tests/conf/chabud.yaml

```diff
@@ -0,0 +1,18 @@
+model:
+  class_path: SemanticSegmentationTask
+  init_args:
+    loss: "ce"
+    model: "unet"
+    backbone: "resnet18"
+    in_channels: 6
+    num_classes: 2
+    num_filters: 1
+    ignore_index: null
+data:
+  class_path: ChaBuDDataModule
+  init_args:
+    batch_size: 2
+  dict_kwargs:
+    root: "tests/data/chabud"
+    bands: ["B4", "B3", "B2"]
+    download: true
```

Comment:
Can you add an EOL char? Probably easiest to configure your editor to do this automatically.